docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,252 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+server statistics on the current connected WBEM server  and the return of
+server response time by setting the GatherStatisticalData property to False in
+the CIM_ObjectManager instance in the Interop namespace. See '_Operation
+statistics'_ for more information on statistics in pywbem/pywbemcli and WBEM
+servers.
+
+Since only some WBEM servers  actually implement statistics gathering, this
+request may result in an exception from the server because the Class
+CIM_ObjectManager or its property ``GatherStatisticalData`` do not exist in the
+server or the server does not allow a client to modify the property.
+
+Once the WBEM server stops returning the server response time, the  column
+``Server Optimes`` no longer appears in the statistics report.
+
+
+.. index:: pair: statistics commands; statistics status
+
+.. _`Statistics status command`:
+
+``statistics status`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics status`` command attempts display the status of both the
+statistic gathering in the current WBEM server and in pywbemcli.
+
+See '_Operation statistics'_ for
+more information on statistics in pywbem/pywbemcli and WBEM servers
+
+.. index:: pair: statistics commands; statistics reset
+
+.. _`Statistics reset command`:
+
+``statistics reset`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics reset`` command attempts display the status of both the
+statistic gathering in the current WBEM server and in pywbemcli.
+
+.. index:: pair: statistics commands; statistics show
+
+.. _`Statistics show command`:
+
+``statistics show`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics show`` command displays the current pywbemcli managed
+statistics as a table if pywbemcli time statistics is enabled with
+``--timestats`` general options
+
+The following example shows the use of --timestats with the ``cmd`` option in
+the interactive mode executing ``statistics show``.
+
+
+.. code-block:: text
+
+    $ pywbemcli -n mock1 --timestats cmd
+
+    pywbemcli> class enumerate --di --no
+    CIM_Foo
+    CIM_Foo_sub
+    CIM_Foo_sub2
+    CIM_Foo_sub_sub
+     pywbemcli> statistics show
+
+    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
+       Op    Exc     Op Time(S)  Operation
+      Cnt    Cnt    Avg/Min/Max
+    -----  -----  -------------  -------------------
+        1      0          0.000  EnumerateClassNames
+
+
+.. index:: pair: statistics commands; statistics server-show
+
+.. _`Statistics server-show command`:
+
+``statistics server-show`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``statistics server-show`` command displays the current WBEM server gathered
+statistics. What is returned depends on the implementation of statistics
+gathering in the WBEM server.
+
+This command attempts to display the server statistics. It does that by
+retrieving "CIM_COMOMStatisticalData" instances associated from the WBEM server
```

Comment:
Typo in class name: "CIM_COMOMStatisticalData"

Reply:
DONE